sonic_platform_base/sonic_xcvr/sfp_optoe_base.py

```diff
@@ -47,6 +55,18 @@ def is_coherent_module(self):
         api = self.get_xcvr_api()
         return api.is_coherent_module() if api is not None else None
 
+    def get_transceiver_vdm_real_value(self):
```

Comment:
@mihirpat1 can you add comments what real means

Reply:
@prgeor I have addressed this now